python/instrumentation/openinference-instrumentation-beeai/src/openinference/instrumentation/beeai/__init__.py

```diff
@@ -28,7 +28,7 @@
 
 from .middleware import create_telemetry_middleware
 
-_instruments = ("beeai-framework >= 0.1.10, < 0.1.17",)
+_instruments = ("beeai-framework >= 0.1.10, < 0.1.18",)
```

Comment:
a lot of instrumentation (most?) don't put an upper bound as doing so is continual maintenance (basically revlock until someone makes a PR like this and then it gets released. I ran into revlock here and would run into again immediately as 0.1.18 is out now

Reply:
Unfortunately, the beeai framework is developing very quickly, and we haven't released the first major version yet. This means that the framework does not yet use standard semantic versioning. The braking changes can be hidden in the patch update. This is the reason why we are testing each new published version.
When we use the framework and the instrumentation, we always use the specific fixed version.

Reply:
Fair enough as long as leads here can turn around releases as you make them.

Do you mind looking at the PR i made? There were a couple lints i noticed and some places where versions were not coherent. I'm happy to close mine out as long as yours works with today's version of beeai!

Reply:
I've updated the instrumentation using the current version BeaAI. 
I had a look at your PR and I fixed the `entry-points` names due to your change (thanks). 
We should use the version range until we release the first major version of BeeAI.